policyengine_us/parameters/gov/states/mt/tax/income/exemptions/amount.yaml

```
@@ -0,0 +1,15 @@
+description: Montana credits this amount for each eligible exemption.
```

Some states provide an exemption credit but Montana's is the more standard deduction
```suggestion
description: Montana deducts this amount for each eligible exemption when computing taxable income.
```